app/controllers/api/connections_controller.rb

```diff
@@ -47,6 +47,7 @@ def request_match
     end
 
     def update_match
+        logger = Rails.logger
```

Comment:
please double check, I think logger is already global. If it isn't, it would be nice to understand how to make it global. Otherwise we will be creating new instances everytime the code runs which is very good.

Reply:
This is awkward. Turns out "logger" is already global, but when I tried it, it threw an error. Don't know what happened but this line is not necessary at all